cluster/charts/crossplane-types/templates/clusterrole.yaml

```diff
@@ -66,15 +62,8 @@ rules:
   - list
   - watch
 - apiGroups:
-  - cache.crossplane.io
-  - compute.crossplane.io
-  - database.crossplane.io
   - kubernetes.crossplane.io
```

Comment:
I remember a PR removing Kubernetes Provider type usage from other repos. Do we need to keep it here?

Reply:
I don't think so; I can remove it.